cloud/pkg/dynamiccontroller/application/application.go

```diff
@@ -95,6 +95,7 @@ type Application struct {
 
 	count     uint64 // count the number of current citations
 	countLock sync.Mutex
+	tim       time.Time // record the increase and decrease time of application instances
```

Comment:
better `tim // record the last closing time of application, only make sense when count ==0`
and add a func
```go
func(app *Application) LastCloseTime() tme.Time{
    app.countLock.Lock
    defer app.countLock.Unlock
    if app.count ==0 && app.lastCloseTime != nil {
        return app.lastCloseTIme
    }
    return time.Time{}
}

Reply:
fixed